app/views/obituaries/show.html.erb

```diff
@@ -9,3 +9,6 @@
 <h2> State: <%= @obituary.state %> </h2>
 
 <p><%= @obituary.description %></p>
+
+<p><%= link_to 'Edit', edit_obituary_path(@obituary.id) %></p>
```

Comment:
Oops, I still need to wrap this in a conditional to make sure the user can only see these links if the obituary belongs to them.